### DISCUSSION TITLE
Desktop-v0.0.7

### DISCUSSION THREAD

**Discussion by SWHL** [SWHL/RapidVideOCR#38] [Announcements]
- 修复`v0.0.6`中打开闪退问题
- 日常运行日志会默认存储在`RapidVideOCR-Desktop-v0.0.7-windows-2022-X64-no-installer\RapidVideOCR\rapid_videocr\log`下，默认保存5天，过期会自动删除。

<hr /><em>This discussion was created from the release <a href='https://github.com/SWHL/RapidVideOCR/releases/tag/Desktop-v0.0.7'>Desktop-v0.0.7</a>.</em>

**Comment by Ebridger26** [comment]
Is there a way to have this software in English?

**Reply by SWHL** [reply]
Please attention to this repo: https://github.com/SWHL/RapidVideOCRDesktop

**Comment by Ebridger26** [comment]
I will.Brian ***@***.***/shop/HardbackYoYowww.hardbackyoyo.comOn Apr 11, 2024, at 7:12 PM, SWHL ***@***.***> wrote:﻿
Please attention to this repo: https://github.com/SWHL/RapidVideOCRDesktop

**Reply by SWHL** [reply]
I have released v0.0.8, and add the English version. See details for [link](https://github.com/SWHL/RapidVideOCRDesktop/releases/tag/v0.0.8)

I have a request. Is it convenient for you to polish your English? After all, my native language is not English.

**Comment by Ebridger26** [comment]
What do you mean by “polish my English”? English is my first language?Brian ***@***.***/shop/HardbackYoYowww.hardbackyoyo.comOn Apr 11, 2024, at 8:12 PM, SWHL ***@***.***> wrote:﻿
I have released v0.0.8, and add the English version. See details for link
I have a request. Is it convenient for you to polish your English? After all, my native language is not English.

**Comment by Ebridger26** [comment]
I don’t know which file to open in the folder. 

I downloaded
RapidVideOCR-v0.0.8-windows-2022-X64-no-installer_en.zip, and I opened the RapidVideOCR.exe file. A command prompt briefly popped up on screen, and it then closed immediately.
 <https://github.com/SWHL/RapidVideOCRDesktop/releases/download/v0.0.8/RapidVideOCR-v0.0.8-windows-2022-X64-no-installer_en.zip>

Brian Martin
847-807-9111
***@***.***
www.etsy.com/shop/HardbackYoYo
www.hardbackyoyo.com

**Comment by Ebridger26** [comment]
我不知道要打开 RapidVideOCR 文件夹中的哪个文件才能打开实际的软件程序。

RapidVideOCR-v0.0.8-windows-2022-X64-no-installer_en.zip
 <https://github.com/SWHL/RapidVideOCRDesktop/releases/download/v0.0.8/RapidVideOCR-v0.0.8-windows-2022-X64-no-installer_en.zip>

我打开了 RapidVideOCR.exe 文件，因为我认为就是这样。但是，屏幕上短暂弹出命令提示符，然后立即关闭。我不知道该怎么办。

Brian Martin
847-807-9111
***@***.***
www.etsy.com/shop/HardbackYoYo
www.hardbackyoyo.com

**Comment by SWHL** [comment]
What I mean is that I would like you to help me make the English version of RapidVideOCR more suitable for your use. After all, English is not my native language.

---- Replied Message ----
| From | ***@***.***> |
| Date | 04/13/2024 04:02 |
| To | SWHL/RapidVideOCR ***@***.***> |
| Cc | SWHL ***@***.***>,
State ***@***.***> |
| Subject | Re: [SWHL/RapidVideOCR] Desktop-v0.0.7 (Discussion #38) |

What do you mean by “polish my English”? English is my first language?Brian ***@***.***/shop/HardbackYoYowww.hardbackyoyo.comOn Apr 11, 2024, at 8:12 PM, SWHL ***@***.***> wrote:﻿
I have released v0.0.8, and add the English version. See details for link
I have a request. Is it convenient for you to polish your English? After all, my native language is not English.

**Comment by SWHL** [comment]
Firstly, it is recommended to try the Chinese version of v0.0.7 first. Double click to open RapidVideOCR.exe and observe if there is any flashing issue.

Secondly, if it still crashes, can you help me record the operation process? I would like to take a closer look at what error the flashing terminal reported. After all, everyone's computer environment is different.

Thanks.

**Comment by Ebridger26** [comment]
﻿﻿This is the Chinese version.

**Reply by SWHL** [reply]
Can you record the operation process when you click the RapidVideOCR.exe?
This way I can see where the problem is.

Or you can directly use the `rapid_videocr` package. See details for [link](https://swhl.github.io/RapidVideOCR/docs/tutorial/senior/)

**Comment by SWHL** [comment]
I need to see what happened when you double click exe.

**Comment by Ebridger26** [comment]
I showed you what happened when I double clicked on it in that video.Brian ***@***.***/shop/HardbackYoYowww.hardbackyoyo.comOn Apr 12, 2024, at 9:12 PM, SWHL ***@***.***> wrote:﻿
Can you record the operation process when you click the RapidVideOCR.exe?
This way I can see where the problem is.
Or you can directly use the rapid_videocr package. See details for link

**Reply by SWHL** [reply]
Sorry, I didn't see where the video is. 😂😂

**Comment by Ebridger26** [comment]
I embedded the video in an email I sent an hour and a half ago.

Brian Martin
847-807-9111
***@***.***
www.etsy.com/shop/HardbackYoYo
www.hardbackyoyo.com

**Reply by SWHL** [reply]
Sorry, I didn't find it. Maybe it's a network problem here. Can you send it again?
You can send it to my email. My email is liekkaskono@163.com
